doc/requirements.txt

```diff
@@ -11,4 +11,4 @@ sphinxcontrib-htmlhelp==2.0.1
 sphinxcontrib-serializinghtml==1.1.5
 sphinx-copybutton
 sphinx-gallery==0.10.1
-xanadu-sphinx-theme==0.10.0
+xanadu-sphinx-theme==0.11.0

```

Comment:
@AndrewGardhouse this is also in a couple of mode files, `setup.py` and the root level `requirements.txt` we can update the version there as well